api/cron/takedown_unpublished_models.py

```diff
@@ -18,11 +20,44 @@
 from utils.deployer import ModelDeployer  # # noqa isort:skip
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--task_code",
+        type=str,
+        default="",
+        help=("Task code to filter models by " "when choosing to delete "),
```

Comment:
Technically, the code doesn't just filter for models with this task code. It also takes down all models for every task, if they are unpublished.